_data/simple-icons.json

```diff
@@ -1669,8 +1669,11 @@
             "title": "DHL",
             "hex": "FFCC00",
             "source": "https://www.dpdhl-brands.com/dhl/en/guides/design-basics/logo-and-claim.html"
-        },
-        {
+        }, {
```

Comment:
Hey @service-paradis, this could turn in a mess after lot of additions. Check that each brace is created in a different line. Maybe would be worth add some linting to avoid this... :thinking:

Reply:
Oh, you're right! I missed this one, sorry!
In fact, I though there was a linting rule for this. I think it could be worth it to add!

Reply:
> In fact, I though there was a linting rule for this.

Me too.

Reply:
Fixed in #4318

Reply:
I've opened a pull to add a test that checks the prettification of the file.